android-lib/src/main/java/com/kinvey/android/callback/KinveyUserCallback.kt

```diff
@@ -28,7 +28,7 @@ import com.kinvey.java.store.UserStoreRequestManager
 interface KinveyUserCallback<T : User> : KinveyClientCallback<T> {
 
 
-    override fun onSuccess(result: T)
+    override fun onSuccess(result: T?)
```

Comment:
success responses should always have a value, if the response is `void` I believe the `Nothing` Kotlin keyword can be used

Reply:
Removed nullable `T?` type here.
Added for code where entity can be `null` some additional checks.